src/compiler/builder.ts

```diff
@@ -1039,7 +983,7 @@ function handleDtsMayChangeOfFileAndExportsOfFile(
 function getSemanticDiagnosticsOfFile(
     state: BuilderProgramStateWithDefinedProgram,
     sourceFile: SourceFile,
-    cancellationToken: CancellationToken | undefined,
+    cancellationToken?: CancellationToken,

```

Comment:
Is this intentional? IIRC the token is usually required as to not forget to pass it around.

Reply:
Good catch.. that got left out from WIP and fiddling where and how to get the errors